### PR TITLE
Improve navigation across all documentation

### PR DESCRIPTION
**Pull request by pietrushnic**
Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,18 +2,36 @@
...
-customizable to your tastes and needs.
+Dasharo is open source firmware distribution focusing on clean and simple code,
+long-term maintenance, transparent validation, privacy-respecting
+implementation, liberty for the owners, and trustworthiness for all.
 
-Dasharo is a set of productized services, Open Core, and SaaS products which
-help to provide scalable, modular, easy to combine Open Source BIOS, UEFI, and
-Firmware solutions. It offers the components that are needed to develop and
+Dasharo consist of productized services, open-core, and SaaS products which
+help to provide scalable, modular, easy to combine open-source BIOS, UEFI, and
```
Comment (mkopec) [inline docs/index.md:10]: We use `open-source` here but `open source` in line 5

File: docs/ecosystem/overview.md [hunk 2]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
-* [Signing Server](signing-server.md)
-* [Attestation Server](attestation-server.md)
-* [Deployment and Provisioning Server](deployment-and-provisioning-server.md)
-* [Reference OS](reference-os.md)
-* [Publishing Server](publishing-server.md)
-* [End User Device](end-user.md)
+* Open Source Firmware Community - semi-organised collections of contributors
+  to projects related to embedded firmware. Dasharo project contributors are
+  part of OSF community.
+* Development - Dasharo Development describe processes which Embedded Firmware
```
Comment (mkopec) [inline docs/ecosystem/overview.md:14]:
```suggestion
* Development - Dasharo Development describes processes which Embedded Firmware
```

File: docs/ecosystem/overview.md [hunk 3]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  Team use for development of Dasharo Modules, which are sometimes custom
```
Comment (mkopec) [inline docs/ecosystem/overview.md:15]:
```suggestion
  Team uses for development of Dasharo Modules, which are sometimes custom
```

File: docs/ecosystem/overview.md [hunk 4]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  services, but most of the time ready to use productised services, Open Core
+  or SaaS.
+* Production - Dasharo Production describe process of putting together Dasharo
```
Comment (mkopec) [inline docs/ecosystem/overview.md:18]:
```suggestion
* Production - Dasharo Production describes the process of putting together Dasharo
```

File: docs/ecosystem/overview.md [hunk 5]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  Modules according to specific requirements. In this section we describe
+  development infrastructure, build environment, CI/CD and build publishing
+  methods. We also cover how production interact with Dasharo Transparent
```
Comment (mkopec) [inline docs/ecosystem/overview.md:21]:
```suggestion
  methods. We also cover how production interacts with Dasharo Transparent
```

File: docs/ecosystem/overview.md [hunk 6]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  Validation System.
+* Customer Infrastructure - Some customers may have need of replicating part of
+  Dasharo Production process and infrastructure in their own environment, what
```
Comment (mkopec) [inline docs/ecosystem/overview.md:24]:
```suggestion
  Dasharo Production process and infrastructure in their own environment, which
```

File: docs/ecosystem/overview.md [hunk 7]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  leads to repositories synchronization, different CI/CD setup and other needs.
+  This section describe interaction with Customer Infrastructure and what part
```
Comment (mkopec) [inline docs/ecosystem/overview.md:26]:
```suggestion
  This section describes interaction with Customer Infrastructure and which part
```

File: docs/ecosystem/overview.md [hunk 8]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  of Dasharo Production is deployable on customer site.
+* Transparent Validation - Dasharo Transparent Validation (DTV) is set of
```
Comment (mkopec) [inline docs/ecosystem/overview.md:28]:
```suggestion
* Transparent Validation - Dasharo Transparent Validation (DTV) is a set of
```

File: docs/ecosystem/overview.md [hunk 9]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  software, firmware and hardware components with the goal of simplifying and
+  improving long term maintenance of Dasharo firmware. In this section we
+  introduce components, which belong to DTV.
```
Comment (mkopec) [inline docs/ecosystem/overview.md:31]:
```suggestion
  introduce components which belong to DTV.
```

File: docs/index.md [hunk 10]
```diff
@@ -2,18 +2,36 @@
 
 ## About
 
-A new kind of BIOS firmware that will solve the problems of ownership, security
-and compatibility allowing you create secure, efficient images that are fully
-customizable to your tastes and needs.
+Dasharo is open source firmware distribution focusing on clean and simple code,
```
Comment (pietrushnic) [inline docs/index.md:5]:
```suggestion
Dasharo is open-source firmware distribution focusing on clean and simple code,
```

File: docs/index.md (hunk 1 above)
Comment (pietrushnic) [reply docs/index.md:10]: You're right - fixed that.

**Comment by pietrushnic** [comment]
@mkopec thanks for corrections, I forgot to apply grammarly to that text.

**Review by mkopec — APPROVED** [review]

File: docs/ecosystem/overview.md [hunk 11]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
-* [Attestation Server](attestation-server.md)
-* [Deployment and Provisioning Server](deployment-and-provisioning-server.md)
-* [Reference OS](reference-os.md)
-* [Publishing Server](publishing-server.md)
-* [End User Device](end-user.md)
+* Open Source Firmware Community - semi-organised collections of contributors
+  to projects related to embedded firmware. Dasharo project contributors are
+  part of OSF community.
+* Development - Dasharo Development describes processes which Embedded Firmware
+  Team uses for development of Dasharo Modules, which are sometimes custom
```
Comment (sulewskiprzemyslaw) [inline docs/ecosystem/overview.md:15]:
```suggestion
  Team uses for the development of Dasharo Modules, which are sometimes custom
```

File: docs/ecosystem/overview.md [hunk 12]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+* Production - Dasharo Production describes the process of putting together Dasharo
+  Modules according to specific requirements. In this section we describe
+  development infrastructure, build environment, CI/CD and build publishing
+  methods. We also cover how production interacts with Dasharo Transparent
+  Validation System.
+* Customer Infrastructure - Some customers may have need of replicating part of
+  Dasharo Production process and infrastructure in their own environment, which
+  leads to repositories synchronization, different CI/CD setup and other needs.
+  This section describes interaction with Customer Infrastructure and which part
+  of Dasharo Production is deployable on customer site.
```
Comment (sulewskiprzemyslaw) [inline docs/ecosystem/overview.md:27]:
```suggestion
  of Dasharo Production is deployable on the customer site.
```

File: docs/ecosystem/overview.md [hunk 13]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+* Development - Dasharo Development describes processes which Embedded Firmware
+  Team uses for development of Dasharo Modules, which are sometimes custom
+  services, but most of the time ready to use productised services, Open Core
+  or SaaS.
+* Production - Dasharo Production describes the process of putting together Dasharo
+  Modules according to specific requirements. In this section we describe
+  development infrastructure, build environment, CI/CD and build publishing
+  methods. We also cover how production interacts with Dasharo Transparent
+  Validation System.
+* Customer Infrastructure - Some customers may have need of replicating part of
```
Comment (sulewskiprzemyslaw) [inline docs/ecosystem/overview.md:23]:
```suggestion
* Customer Infrastructure - Some customers may have the need of replicating part of
```

File: docs/ecosystem/overview.md [hunk 14]
```diff
@@ -8,37 +8,38 @@ organization. This section overview role of every component.
...
+  leads to repositories synchronization, different CI/CD setup and other needs.
+  This section describes interaction with Customer Infrastructure and which part
+  of Dasharo Production is deployable on customer site.
+* Transparent Validation - Dasharo Transparent Validation (DTV) is a set of
+  software, firmware and hardware components with the goal of simplifying and
+  improving long term maintenance of Dasharo firmware. In this section we
+  introduce components which belong to DTV.
+* Testing Infrastructure - Part of Dasharo Transparent Validation responsible
+  for scheduling automated testing, publishing test results and interacting
+  with Dasharo Production in area of release candidate quality confirmation.
```
Comment (sulewskiprzemyslaw) [inline docs/ecosystem/overview.md:34]:
```suggestion
  with Dasharo Production in the area of release candidate quality confirmation.
```

File: docs/index.md [hunk 15]
```diff
@@ -23,6 +41,12 @@ of your platform.
   source hardware and open instruction set architecture. Feel free to check the
```
Comment (sulewskiprzemyslaw) [inline docs/index.md:41]: If we in all documents use the term `open-source`, I think that we should also change the `open source` in line 40.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

**Review by sulewskiprzemyslaw — APPROVED** [review]

File: docs/index.md (hunk 15 above)
Comment (sulewskiprzemyslaw) [reply docs/index.md:41]: I went through all the documentation. All occurrences of the phrase `open-source` (excluding proper names) has been fixed. I have no further comments.

**Pull request merged**